pkg/util/httpproxy/upgradeawareproxy.go

```diff
@@ -35,11 +36,13 @@ func NewUpgradeAwareSingleHostReverseProxy(clientConfig *kclient.Config, backend
 	if err != nil {
 		return nil, err
 	}
+	reverseProxy := httputil.NewSingleHostReverseProxy(backendAddr)
+	reverseProxy.FlushInterval = 50 * time.Millisecond
```

Comment:
The interval is reasonable. The downside is that net/http/httputil seems to wake up every 50 milliseconds:

http://golang.org/src/net/http/httputil/reverseproxy.go?s=2744:2819#L207